src/SolutionTemplate/Uno.ProjectTemplates.Dotnet/content/unoapp/.template.config/template.json

```diff
@@ -2,250 +2,282 @@
-      "defaultValue": "false"
-    }
+	"guid1": {
+	  "type": "generated",
+	  "generator": "guid",
+	  "replaces": "$guid1$",
+	  "parameters": {
+		"format": "N"
+	  }
+	},
+	"guid2": {
+	  "type": "generated",
+	  "generator": "guid",
+	  "replaces": "$guid2$",
+	  "parameters": {
+		"format": "N"
+	  }
+	},
+	"guid3": {
+	  "type": "generated",
+	  "generator": "guid",
+	  "replaces": "$guid3$",
+	  "parameters": {
+		"format": "N"
+	  }
+	},
+	"guid4": {
+	  "type": "generated",
+	  "generator": "guid",
+	  "replaces": "$guid4$",
+	  "parameters": {
+		"format": "N"
+	  }
+	},
+	"guid5": {
+	  "type": "generated",
+	  "generator": "guid",
+	  "replaces": "$guid5$",
+	  "parameters": {
+		"format": "N"
+	  }
+	},
+	"guid6": {
+	  "type": "generated",
+	  "generator": "guid",
+	  "replaces": "$guid6$",
+	  "parameters": {
+		"format": "N"
+	  }
+	},
+	"guid7": {
+	  "type": "generated",
+	  "generator": "guid",
+	  "replaces": "$guid7$",
+	  "parameters": {
+		"format": "N"
+	  }
+	},
+	"windowsPublisherName": {
+	  "type": "parameter",
+	  "datatype": "text",
+	  "replaces": "$XmlEscapedPublisherDistinguishedName$",
+	  "description": "Provides the value to use for the Windows head publisher",
+	  "defaultValue": "O=MyCompany"
+	},
+	"WebAssembly": {
+	  "type": "parameter",
+	  "dataType": "bool",
+	  "defaultValue": "true",
+	  "description": "Enables the WebAssembly platform support project"
+	},
+	"mobile": {
+	  "type": "parameter",
+	  "dataType": "bool",
+	  "defaultValue": "true",
+	  "description": "Enables the Mobile platform support project"
+	},
+	"skia-wpf": {
+	  "type": "parameter",
+	  "dataType": "bool",
+	  "defaultValue": "true",
+	  "description": "Enables the Skia/WPF platform support project"
+	},
+	"skia-gtk": {
+	  "type": "parameter",
+	  "dataType": "bool",
+	  "defaultValue": "true",
+	  "description": "Enables the Skia/GTK platform support project"
+	},
+	"skia-linux-fb": {
+	  "type": "parameter",
+	  "dataType": "bool",
+	  "defaultValue": "true",
+	  "description": "Enables the Skia/Linux Framebuffer platform support project"
+	},
+	"winui-desktop": {
+	  "type": "parameter",
+	  "dataType": "bool",
+	  "defaultValue": "true",
+	  "description": "Enables the WinUI desktop platform support project"
+	},
+	"vscode": {
+	  "type": "parameter",
+	  "dataType": "bool",
+	  "defaultValue": "false",
+	  "description": "Adds the Visual Studio Code Debugging support files for WebAssembly"
+	},
+	"skipRestore": {
+	  "type": "parameter",
+	  "datatype": "bool",
+	  "description": "If specified, skips the automatic restore of the project on create.",
+	  "defaultValue": "false"
+	},
+	"UseAndroid": {
+	  "type": "generated",
+	  "generator": "constant",
+	  "dataType": "bool",
+	  "parameters": {
+		"value": "True"
+	  }
+	},
+	"UseIOS": {
+	  "type": "generated",
+	  "generator": "constant",
+	  "dataType": "bool",
+	  "parameters": {
+		"value": "True"
+	  }
+	},
+	"UseAppKit": {
```

Comment:
Does "UseAppKit" have some historical context? It would be clearer if this was just UseMacOS and would map closer to the actual tfm

Reply:
Yeah, this is linked to the way the VSIX is built, but it's not aligned with the rest, will change. Thanks!